src/extraction/FStar.Extraction.ML.Term.fs

```diff
@@ -1263,6 +1263,10 @@ and term_as_mlexpr' (g:uenv) (top:term) : (mlexpr * e_tag * mlty) =
             | _ -> term_as_mlexpr g t
          end
 
+        | Tm_meta (t, Meta_monadic_lift (m1, _m2, _ty))
```

Comment:
nice and clean

Reply:
Can we add a comment here explaining what guarantees that that _ty is erasable?
It has to do with lifts from erasable to non-erasable effects enforcing a non-informativeness of the type.